cni/network/network.go

```diff
@@ -35,13 +40,36 @@ const (
 	CNI_UPDATE = "UPDATE"
 )
 
+const (
+	// URL to query NMAgent version and determine whether we snat on host
+	nmAgentSupportedApisURL = "http://168.63.129.16/machine/plugins/?comp=nmagent&type=GetSupportedApis"
+	// Only SNAT support (no DNS support)
+	nmAgentSnatSupportAPI = "NetworkManagementSnatSupport"
+	// SNAT and DNS are both supported
+	nmAgentSnatAndDnsSupportAPI = "NetworkManagementDnsSupport"
```

Comment:
NetworkManagementDnsSupport - Does this signify both snat and dns support? Did you check with nmagent?

Reply:
yes, I've verified with them